test/testenv/appframework_utils.go

```diff
@@ -449,3 +449,47 @@ func AppFrameWorkVerifications(ctx context.Context, deployment *Deployment, test
 	}
 	return clusterManagerBundleHash
 }
+
+// GetIsDeploymentInProgressFlag returns IsDeploymentInProgress for given CR Name, CR Kind
+func GetIsDeploymentInProgressFlag(ctx context.Context, deployment *Deployment, testenvInstance *TestCaseEnv, name string, crKind string) (bool, error) {
+	var isDeploymentInProgress bool
+	var err error
+	switch crKind {
+	case "Standalone":
```

Comment:
indexercluster missing ?

Reply:
IndexerCluster does not have app framework spec but I think LicenseManager is missing here.

Reply:
The flag is only set on CR's which accept Appframework spec, hence skipping idxc.